mailscanner/conf.php.example

```diff
@@ -145,7 +145,8 @@ define('SHOW_MORE_INFO_ON_REPORT_GRAPH', false);
 /* Mailwatch Interface Settings */
 define('MAILWATCH_MAIL_HOST', '127.0.0.1');
 define('MAILWATCH_MAIL_PORT', '25');
-define('MAILWATCH_FROM_ADDR', 'postmaster');
+// Change for your own email address
```

Comment:
postmaster@domain.tld if fine for me, it's the commented description that can be confusing: to I have to put my very own email address or a generic corporate email like postmaster@domain.tld?